vnext/ReactWindowsCore/Modules/StatusBarManagerModule.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "StatusBarManagerModule.h"
+#include <VersionHelpers.h>
```

Comment:
Nit: I can remove this header.

Reply:
Yes, please remove: it seems out of place.

---
In reply to: [408908658](https://github.com/microsoft/react-native-windows/pull/4609#discussion_r408908658) [](ancestors = 408908658)